File: frontend/src/types/branddeal.ts
export interface BrandDeal {
  id: number
  brandName: string
  contactName?: string
  contactEmail?: string
  dealValue?: number
  currency: string
  status: string
  deadline?: string
  deliverables: string[]
  notes?: string
  createdAt?: string
}

export interface MediaKit {
  id: number
  displayName: string
  bio?: string
  categories: string[]
  socialLinks: Record<string, string>
  rateCard: Record<string, any>
  isPublic: boolean
  slug?: string
  createdAt?: string
}

export type DealStatus = 'INQUIRY' | 'NEGOTIATION' | 'CONTRACTED' | 'IN_PROGRESS' | 'COMPLETED' | 'CANCELLED'

File: backend/onGo-domain/src/main/kotlin/com/ongo/domain/branddeal/BrandDeal.kt
package com.ongo.domain.branddeal

import java.time.LocalDate
import java.time.LocalDateTime

data class BrandDeal(
    val id: Long? = null,
    val userId: Long,
    val brandName: String,
    val contactName: String? = null,
    val contactEmail: String? = null,
    val dealValue: Long? = null,
    val currency: String = "KRW",
    val status: String = "INQUIRY",
    val deadline: LocalDate? = null,
    val deliverables: String? = "[]",
    val notes: String? = null,
    val createdAt: LocalDateTime? = null,
    val updatedAt: LocalDateTime? = null,
)
